src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -172,14 +172,13 @@
          (get-in ws ["workspace" "bucketName"])])
       (style/create-section-header "Analysis Submissions")
       (style/create-paragraph
-        (let [count-all (count submissions)
-              grouped-by-status (group-by #(% "status") submissions)]
+        (let [count-all (reduce + 0 (vals submissions-count))]
              [:div {}
               (str count-all " Submission" (when-not (= 1 count-all) "s"))
               (when (pos? count-all)
                     [:ul {:style {:marginTop "0"}}
-                     (for [[status subs] grouped-by-status]
-                        [:li {} (str (count subs) " " status)])])]))]]))
+                     (for [[status subs] (sort (seq submissions-count))]
```

Comment:
Does this `seq` do anything?  Looks like it should already be a vector.  And I suppose you're sorting just for consistency's sake?